LPD-62179

victorg1991 opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

victorg1991 committed:
LPD-62179 This has the correct type already

When enableObjectEntryVersioning is enabled in the object it creates a new version when publishing. This version is using the object entry dto as a base which already have the correct type for the Picklist and multiselectPicklist. Without this it tries to cast it to the wrong type and fail

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
